### PR TITLE
Dump new API

### PR DESCRIPTION
**Pull request by igordmn**

### CODE REVIEW

File: compose/ui/ui/api/desktop/ui.api [hunk 1]
```diff
@@ -3119,6 +3119,24 @@ public final class androidx/compose/ui/platform/LocalSoftwareKeyboardController
 	public final fun provides (Landroidx/compose/ui/platform/SoftwareKeyboardController;)Landroidx/compose/runtime/ProvidedValue;
 }
 
+public final class androidx/compose/ui/platform/PlatformInsets {
```
Comment (igordmn) [inline compose/ui/ui/api/desktop/ui.api:3122]: There was a mistake on CI, and it didn't run the checks. I fixed it here:
https://teamcity.jetbrains.com/admin/settingsDiffView.html?id=buildType:JetBrainsPublicProjects_Compose_DesktopTestsForPullRequests&versionBefore=49&versionAfter=50&actionId=1756398

Now it run API checks properly.

I merge it now to fix the CI. If anything needs to be adressed, let's do that in the next PR's

File: compose/ui/ui/api/desktop/ui.api [hunk 2]
```diff
@@ -3202,11 +3220,16 @@ public final class androidx/compose/ui/platform/ViewConfiguration$DefaultImpls {
 }
 
 public abstract interface class androidx/compose/ui/platform/WindowInfo {
-	public abstract fun getContainerSize-YbymL2g ()J
-	public abstract fun getKeyboardModifiers-k7X9c1A ()I
+	public fun getContainerSize-YbymL2g ()J
```
Comment (igordmn) [inline compose/ui/ui/api/desktop/ui.api:3223]: It is the correct ABI, as it has a default implementation. The dump was dumped when we had incorrect ABI

File: compose/ui/ui/api/desktop/ui.api [hunk 3]
```diff
@@ -3657,15 +3680,16 @@ public final class androidx/compose/ui/window/DialogProperties {
 	public static final field $stable I
 	public fun <init> (ZZZ)V
 	public synthetic fun <init> (ZZZILkotlin/jvm/internal/DefaultConstructorMarker;)V
-	public synthetic fun <init> (ZZZJILkotlin/jvm/internal/DefaultConstructorMarker;)V
-	public synthetic fun <init> (ZZZJLkotlin/jvm/internal/DefaultConstructorMarker;)V
+	public synthetic fun <init> (ZZZZJILkotlin/jvm/internal/DefaultConstructorMarker;)V
```
Comment (igordmn) [inline compose/ui/ui/api/desktop/ui.api:3683]: This constructor is experimental, but the dump doesn't show that :(

File: MULTIPLATFORM.md [hunk 4]
```diff
@@ -13,7 +13,7 @@
 
 Run tests for Desktop:
 ```bash
-./gradlew :mpp:testDesktop
+./gradlew desktopTest
```
Comment (igordmn) [inline MULTIPLATFORM.md:16]: `testDesktop` was removed [here](https://github.com/JetBrains/compose-multiplatform-core/pull/802#discussion_r1322849760)

**Review by MatkovIvan — APPROVED** [review]

**Pull request merged**